### PR TITLE
Create find_text_all_file_dir.py

### PR DESCRIPTION
**Pull request by yanmo96**
To find certain text from all files under give dir.
This script will remove all space and put all text to lowercase for both search text and files before try to match.

### CODE REVIEW

**Comment by yanmo96** [comment]
To run this script, do : `python Create find_text_all_file_dir.py`

File: tools/find_text_all_file_dir.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+
+user_input = input('What is the name of your directory: ')
```
Comment (xieus) [inline tools/find_text_all_file_dir.py:3]: Can you create one sub-directory under tools and put the script there?

Comment (yanmo96) [reply tools/find_text_all_file_dir.py:3]: Will do👍

Comment (xieus) [reply tools/find_text_all_file_dir.py]: I would recommend renaming the directory to /tools/license/..

**Comment by xieus** [comment]
> To run this script, do : `python Create find_text_all_file_dir.py`

Could you give an example of running this script?

File: tools/find_text_all_file_dir.py (hunk 1 above)
Comment (yanmo96) [reply tools/find_text_all_file_dir.py]: Changed, thanks!

**Comment by yanmo96** [comment]
> 

After do the : `python Create find_text_all_file_dir.py`
The screen will prompt asking for which file to search from, please type in the file/folder's relative location or the full path. 
Then the screen will prompt asking for what to search, then please type in the searching string.
Note: During search, the script will remove space and change to all lower case for both the file and search string. Thus to much as much as we can

**Review by xieus — APPROVED** [review]

**Pull request merged**